stringbuilder.go

```diff
@@ -289,6 +289,19 @@ func (s *StringBuilder) Reverse() *StringBuilder {
 	return s
 }
 
+// Substring creates a substring from the string builder using a start and an end(exclusive) bound
```

Comment:
```suggestion
// Returns a substring from start (inclusive) to end (exclusive).
```

As the comment is above the method-name itself, I don't think we need the name again - plus it fits better to the overall pattern.

Reply:
Done